gulpfile.js

```diff
@@ -28,9 +40,31 @@ function build_lib_sourcemap(){
   .pipe(gulp.dest('lib/'));
 }
 
+// npm compliant build with source-maps
+function build_lib_rsa_sourcemap(){
+  // run webpack
+  return gulp.src('index.js').pipe(webpackStream(createWebpackConfig({
+    mode: 'development',
+    entry: npmEntry,
+    output: {
+        filename:'oidc-client.rsa256.js',
+        libraryTarget:'umd',
+        // Workaround for https://github.com/webpack/webpack/issues/6642
+        globalObject: 'this'
+    },
+    plugins: [
+      new webpack.NormalModuleReplacementPlugin(/(.*)JoseUtil(\.js)?$/, (resource) => {
```

Comment:
This code gets duplicated in the config quite a lot - instead of copy/pasting it, could you please assign it to a variable, much like you did with `optimization`, and then re-use it?

Reply:
@ivantm, thanks for comment!
Fixed.